functionary/core/models/scheduled_task.py

```diff
@@ -150,7 +152,8 @@ def set_schedule(self, crontab_schedule: CrontabSchedule) -> None:
         """
         if self.periodic_task is None:
             self.periodic_task = PeriodicTask.objects.create(
-                name=self.name,
+                name=self.id,
+                description=self.description or self.name,
```

Comment:
What's the motivation for adding the description here?  I don't see it becoming something that we would surface anywhere, as the `PeriodicTask` is an internal celery-beat thing that we're leveraging and not a user facing object.  It doesn't hurt anything to set it, I'm just curious if it there was something that lead you to do that.